tfs/collection.py

```diff
@@ -45,51 +45,58 @@ def __new__(mcs, cls_name, bases, dct: dict):
 
 
 class TfsCollection(metaclass=_MetaTfsCollection):
-    """ Abstract class to lazily load and write TFS files.
+    """
+    Abstract class to lazily load and write TFS files.
 
-    The classes that inherit from this abstract class will be able to define
-    TFS files as readable or writable and read or write them just as attribute
-    access or assignments. All attributes will be read and written as Pandas
-    DataFrames.
+    classes inheriting from this abstract class will be able to define TFS files
+    as readable or writable, and read or write them just as attribute access or
+    assignments. All attributes will be read and written as `TfsDataFrames`.
 
     Example:
-        If `./example` is a directory that contains two TFS file `getbetax.out`
-        and `getbetax.out` with BETX and BETY columns respectively:
+        If **./example** is a directory that contains two TFS files `getbetax.tfs`
+        and `getbetax.tfs` with BETX and BETY columns respectively:
```

Comment:
Put a real file names we use, for example, beta_phase_x.tfs and beta_phase_y.tfs